styleguide/derek/examples/stories-interior.ejs

```diff
@@ -0,0 +1,49 @@
+<%- partial("../_partials/_ceiling") %>
+<%- partial("../_partials/_mainnav") %>
+<%- partial('../_partials/solutions/header-smallIMAC', { headline: 'Customer Stories'}) %>
+
+<!-- Story Region -->
+<div class="container standard-padding">
+  <div class="row">
+    <div class="story-colLeft">
+      <h1 class="story-title">Under Armour</h1>
+      <p>The company is the developer, marketer, and supplier of a wide range of sportswear and casual apparel mainly focusing on hi-tech sportswear for professional athletes. Under Armour began offering footwear in 2006, and now operates as a global company with offices in Denver, Amsterdam, Denver, Hong Kong, Toronto and Guangzhou, China.</p>
+      <p><a href="javascript: void(0);" class="story-caseStudy-button">Read the Case Study</a></p>
```

Comment:
I thought we were avoiding a third part of maintainable classes. So it would be story-caseStudyButton, unless the third part was a state.